src/test/java/de/mediathekview/mserver/crawler/mdr/tasks/MdrFilmTaskTest.java

```diff
@@ -0,0 +1,129 @@
+package de.mediathekview.mserver.crawler.mdr.tasks;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import de.mediathekview.mlib.daten.Film;
+import de.mediathekview.mlib.daten.GeoLocations;
+import de.mediathekview.mlib.daten.Sender;
+import de.mediathekview.mlib.messages.listener.MessageListener;
+import de.mediathekview.mserver.base.config.MServerConfigManager;
+import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
+import de.mediathekview.mserver.crawler.basic.CrawlerUrlDTO;
+import de.mediathekview.mserver.crawler.mdr.MdrCrawler;
+import de.mediathekview.mserver.progress.listeners.SenderProgressListener;
+import de.mediathekview.mserver.testhelper.AssertFilm;
+import de.mediathekview.mserver.testhelper.JsoupMock;
+import de.mediathekview.mserver.testhelper.WireMockTestBase;
+import java.io.IOException;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Set;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.ForkJoinPool;
+import org.jsoup.Jsoup;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.modules.junit4.PowerMockRunnerDelegate;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Jsoup.class})
+@PowerMockRunnerDelegate(Parameterized.class)
+@PowerMockIgnore("javax.net.ssl.*")
+public class MdrFilmTaskTest extends WireMockTestBase {
+
+  private String requestUrl;
+  private String htmlFile;
+  private String xmlUrl;
+  private String xmlFile;
+  private String expectedTopic;
+  private String expectedTitle;
+  private String expectedDescription;
+  private LocalDateTime expectedTime;
+  private Duration expectedDuration;
+  private String expectedWebsite;
+  private String expectedUrlSmall;
+  private String expectedUrlNormal;
+  private String expectedUrlHd;
+  private String expectedSubtitle;
+  private GeoLocations expectedGeoLocation;
+
+  @Parameterized.Parameters
+  public static Collection<Object[]> data() {
+    return Arrays.asList(
+        new Object[][]{
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)